### ISSUE TITLE
Clarification of Num_Dx2 and Meduse_2 numerators

### ISSUE THREAD

**Issue by DarrenWHenderson** [i2b2plugins/loyalty_cohort#3]
Starting here https://github.com/i2b2plugins/loyalty_cohort/blob/8bbd4d2e542db4040806fb5a357014cf19f6f996/usp_LoyaltyCohort_v6.sql#L136

https://github.com/i2b2plugins/loyalty_cohort/blob/8bbd4d2e542db4040806fb5a357014cf19f6f996/usp_LoyaltyCohort_v6.sql#L137

https://github.com/i2b2plugins/loyalty_cohort/blob/8bbd4d2e542db4040806fb5a357014cf19f6f996/usp_LoyaltyCohort_v6.sql#L147

Can we clarify what the desired numerator for num_dx2 is (this logic is also used to calculate meduse_2 https://github.com/i2b2plugins/loyalty_cohort/blob/8bbd4d2e542db4040806fb5a357014cf19f6f996/usp_LoyaltyCohort_v6.sql#L268)

The comment, and our discussion of the variable in the meetings so far, would imply that it was intended to be a count of patients with multiple diagnoses. As written, it's a count of patients with more than one service dates having a diagnosis fact. The excludes patients that visit multiple clinics in a large hospital system on the same service date for transportation reasons, or scheduling concerns, something we see quite often from our rural patients. But, it also includes patients with the identical diagnosis code that returned for another visit on a different day. The 2nd scenario is less troubling, since we are looking for loyal patients here. 

For meduse_2 though, the second scenario may be more important. IF the intent of the numerator here is patients on multiple drugs, then just counting their service dates is incorrect. It could be the same patient returning to pick up their scrip every 30 days with the same concept_cd and the code counts that patient as meduse_2 = 12.

**Comment by jklann** [comment]
You're correct that our language has been imprecise, but what the code is doing matches what is in Josh's paper. As that was validated in his studies, we're not going to change it right now. But you make a good point.
[Josh supplement table.docx](https://github.com/i2b2plugins/loyalty_cohort/files/6400417/Josh.supplement.table.docx)
